### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by AsalaKM**
**User Profile**
Show user information and all reviews that related to the user.

### CODE REVIEW

**Comment by AsalaKM** [comment]
Relates #5

File: public/js/home.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
+/* global document */
+/* global window */
```
Comment (FarahZaqout) [inline public/js/home.js]: these two lines can be one.
```/* global document window */```

File: src/controllers/user_profile.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const { getUserData } = require('../model/queries/users');
+const { getReviewByUser } = require('../model/queries/users');
+
+exports.get = (({ params: { username } }, res, next) => {
+  getUserData(username)
+    .then((response) => {
+      if (response.length !== 0) {
+        let img;
+        const { gender, username } = response[0];
```
Comment (FarahZaqout) [inline src/controllers/user_profile.js]: ```username``` is already declared in the upperscope. You should not shadow it or modify it.

File: public/js/home.js (hunk 1 above)
Comment (FarahZaqout) [reply public/js/home.js]: also use global fetch

Comment (FarahZaqout) [reply public/js/home.js]: and global alert.

btw alert is generally considered a bad practice. The function alert is regarded as bad user experience, and is generally used in development for debugging purposes. You should use other methods in future projects. You don't need to change the alert now, this is just information to keep in mind in  your next project.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

File: public/js/business_page.js [hunk 3]
```diff
@@ -17,3 +17,28 @@ aboutButton.addEventListener('click', () => {
...
+
+document.querySelectorAll('.result').forEach((element) => {
+  const evaluation = element.querySelector('#evaluation');
+  const oneStar = element.querySelector('#one-star');
+  const twoStar = element.querySelector('#two-star');
+  const threeStar = element.querySelector('#three-star');
+  const fourStar = element.querySelector('#four-star');
+  const fiveStar = element.querySelector('#five-star');
+
+  if (evaluation.textContent >= 1) {
```
Comment (FarahZaqout) [inline public/js/business_page.js:29]: Using switch should be a lot cleaner.

File: src/model/database/db_build.sql [hunk 4]
```diff
@@ -44,4 +44,18 @@ CREATE TABLE comment (
 	content TEXT NOT NULL
 );
 
+INSERT INTO users (email, password, type) VALUES ('asmaa@gmail.com', '000', 'person');
```
Comment (FarahZaqout) [inline src/model/database/db_build.sql]: No inserts in build.sql

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**